src/main/java/com/vinyl/model/Role.java

```diff
@@ -0,0 +1,45 @@
+package com.vinyl.model;
+
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "ROLE")
+public class Role {
+
+	@Id
+	@NotNull
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Integer id;
+	
+	@NotNull
+	@Column(nullable = false)
+	private String roleName;
+
+
+	public Role(@NotNull String roleName) {
```

Comment:
you now don't have a no args construcor which is needed for this class to be a bean, add that constructor, or remove this one if it is not used